cylc/flow/job_runner_mgr.py

```diff
@@ -439,6 +442,12 @@ def _filter_submit_output(cls, st_file_path, job_runner, out, err):
     def _jobs_poll_status_files(self, job_log_root, job_log_dir):
         """Helper 1 for self.jobs_poll(job_log_root, job_log_dirs)."""
         ctx = JobPollContext(job_log_dir)
+        # If the log directory has been deleted prematurely, return a task
+        # failure and an explanation:
+        if not os.path.exists(os.path.join(job_log_root, ctx.job_log_dir)):
+            ctx.run_status = 1
+            ctx.run_signal = 'ERR/' + JOB_FILES_REMOVED_MESSAGE
```

Comment:
I think this would be more conventional. `ERR/` is likely to be confused with the `failed/ERR` message
```suggestion
            ctx.run_signal = 'ERR_JOB_FILES_REMOVED'
```

See
https://github.com/cylc/cylc-flow/blob/3a9cd6a3e17dbb9d794e579d25a83daf62f402e3/cylc/flow/job_runner_mgr.py#L119

Reply:
I think it needs that as a prefix?

Reply:
`ERR` is not a prefix